crates/hyperqueue/src/worker/hwdetect.rs

```diff
@@ -60,6 +61,14 @@ pub fn detect_generic_resource() -> anyhow::Result<Vec<GenericResourceDescriptor
             })
         }
     }
+
+    if let Ok(mem) = read_linux_mem() {
```

Comment:
Let's rename the corresponding function to `detect_generic_resources`, since it now detects more types of resources.